source/kraken/realtime.cpp

```diff
@@ -164,6 +166,14 @@ base_execution_period(const boost::gregorian::date& date, const nt::MetaVehicleJ
     return {boost::posix_time::ptime{date}, boost::posix_time::ptime{date}};
 }
 
+static type::disruption::Message create_message(const std::string& str_msg) {
+    type::disruption::Message msg;
+    msg.text = str_msg;
+    //do we want to add channel types ?
```

Comment:
I guess for now we don't, so we'll just see it when we need it.
This comment line deserves a 'TODO' or to be removed (I prefer removal)